ci-operator/jobs/infra-periodics.yaml

```diff
@@ -29,6 +29,96 @@ periodics:
         requests:
           cpu: 500m
     serviceAccountName: upgrader
+- agent: kubernetes
+  cluster: build09
+  cron: 30 7 * * *
```

Comment:
Could we exclude the weekends?

Reply:
Yes. I would like to apply it to b01's upgrade job too.

Reply:
OK. Changed.